sci-libs/primesieve/primesieve-5.7.3.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="Fast C/C++ prime number generator library"
+DESCRIPTION="primesieve is a program and C/C++ library that generates primes \
+using a highly optimized sieve of Eratosthenes implementation. It counts the \
+primes below 10^10 in just 0.45 seconds on an Intel Core i7-6700 CPU \
+(4 x 3.4GHz). primesieve can generate primes and prime k-tuplets up to 2^64."
+HOMEPAGE="http://primesieve.org/"
+COPYRIGHT="2010-2016, Kim Walisch"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/kimwalisch/primesieve/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="22c5ec34f020592cd559551e3e6e1e86aa51f6e34d639bef352d155ea415ac62"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	primesieve$secondaryArchSuffix = $portVersion
+	cmd:primesieve =  $portVersion
+	lib:libprimesieve$secondaryArchSuffix = 7.0.3 compat >= 7
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgomp$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	primesieve${secondaryArchSuffix}_devel =  $portVersion
+	devel:libprimesieve$secondaryArchSuffix = 7.0.3 compat >= 7
+	"
+REQUIRES_devel="
+	primesieve$secondaryArchSuffix ==  $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	./autogen.sh
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install \
+		 bindir=$commandBinDir
```

Comment:
Well, the packageEntries is a special case: the first argument is the suffix of the sub-package, while all the remaining arguments are paths to files or directories that the packageEntries function needs to move.

Here we could also consider that install is the target we want make to build, and bindir=$commandBinDir is a parameter, but since the merged line is really short we might consider that we don't really need to put it into a separate line.

But you are right. The style is OK both ways. So I'll let you chose the one you prefer.
After all, you are creating the recipe, so you can have the last word for style issues ;-)